src/rgw/rgw_file.h

```diff
@@ -802,7 +802,10 @@ namespace rgw {
 			     const uint32_t flags = RGWFileHandle::FLAG_NONE) {
       using std::get;
 
-      LookupFHResult fhr { nullptr, RGWFileHandle::FLAG_NONE };
+      // This seems like a Clang compiler error.
+      // It result in irgw::RGWFileHandle::FLAG_NONE being undefined during linking.
+      // LookupFHResult fhr { nullptr, RGWFileHandle::FLAG_NONE };
+      LookupFHResult fhr { nullptr, 0 };
```

Comment:
@tchaikov I really needed to read up on this, and then read it again to get the full grasp.
It really looks like this. 

But what I do not understand is that there are more references to RGWFileHandle::FLAG_NONE ( for instance a few lines earlier ), which I suspect are cancelled out, because of the const character of the variable. Where this line does not get the same treatment, and we end up with an undeclared var...
The actual definition looks like:
```
class class RGWFileHandle {
  static constexpr uint32_t FLAG_NONE =    0x0000;
}
```
And per your link-text, that static declaration would require an explicit definition.
But that sort of surpasses the constexpr declaration.

Interested in what your suggestion for a fix is.